tools/view.js

```diff
@@ -440,9 +468,6 @@ View.prototype.displaySuspendEvent = function (data, getSource) {
     var annotationArray = sourceToArray(source.sourceText);
     annotateArray(annotationArray, source.id, data.sections);
     sourceFile.html(arrayToString(annotationArray));
-
-    // enable clicking on EventualSendNodes
-    enableEventualSendClicks(sourceFile);
```

Comment:
It is needed. Currently, when a suspended event is triggered, the whole display of the source file is essentially scraped and recalculated. This also requires reapplying the this part.

At some point, we should avoid doing that, but the idea is that some of the information we want to display is dynamic, so, it can change.